runtests.py

```diff
@@ -38,7 +38,8 @@ def runtests():
     TestRunner = get_runner(settings)
     test_runner = TestRunner(verbosity=1, interactive=True, failfast=True)
     failures = test_runner.run_tests(['bandit', ])
-    sys.exit(failures)
+    if failures:
+        sys.exit(1)
```

Comment:
Can you explain why this change is needed? [sys.exit](https://docs.python.org/3/library/sys.html#sys.exit) returns 1 if `failures` is not an integer, but with the added benefit of printing the failures. Quoting the python docs:  

> If another type of object is passed, None is equivalent to passing zero, and any other object is printed to stderr and results in an exit code of 1.

Reply:
`test_runner.run_tests` returns the number of failures + errors. (https://github.com/django/django/blob/master/django/test/runner.py#L582)

This means that the exit code is unpredictable on failure, this used to be an issue in Django as well (https://code.djangoproject.com/ticket/11615).

Reply:
Interesting! For some reason I thought that `run_tests` returned the actual failure messages. Thanks!